jobbergateapi2/main.py

```diff
@@ -3,11 +3,21 @@
 """
 from fastapi import FastAPI
 from mangum import Mangum
+from starlette.middleware.cors import CORSMiddleware
 
 from jobbergateapi2 import storage
+from jobbergateapi2.config import settings
```

Comment:
We take the path from left to right, so `config` < `routers`, so the order is correct. This projects runs some linters to check that everything is correctly formated, so that kind of problem would break the CI